collectors/conf/storm.conf

```diff
@@ -0,0 +1,6 @@
+[base]
+enabled: false
+interval: 120
```

Comment:
Just for test,  I don't want to print so fast at the terminal. And collecting in 60 second would be great.